tests/Dockerfile

```diff
@@ -52,6 +52,23 @@ RUN dnf -y install \
   && dnf clean all
 RUN echo "%_dbpath $(rpm --eval '%_dbpath')" > /root/.rpmmacros
 
+# Workaround for pkgconf(1)'s unlisted dependency on rpm.
+# This is needed for cmake to work without an rpm installation.
+RUN ln -sf $(rpm --eval '%{_target_platform}%{?_gnu}')-pkg-config \
+           /usr/bin/pkg-config
+
+RUN rpm -e --nodeps --nodb \
```

Comment:
Hey, somebody found a use-case for --nodb :partying_face:

Reply:
Oh! Turns out this is a pretty new feature: 6de0d31ed3b4907dbd7e1683a18ea3a28790614d
So yup, good timing :laughing: